process.py

```diff
@@ -386,9 +387,15 @@ def get_provider_code(directory_name=None):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--path', type=str, default="static/data/lms")
+    specific_folder = raw_input('Do you want to load a specific folder? type y to load a specific folder, type n to load all the data\n')
+    if specific_folder == 'y' or specific_folder == 'Y':
+        dirname = tkFileDialog.askdirectory(initialdir=os.path.abspath("static/data/lms"),  title='Please select a directory')
+        parser.add_argument('--delete_all', dest='delete_all', action='store_true', default=False)
```

Comment:
It's quite confusing to treat the command line option differently depending on the interactive feedback or on another option...
We (the users) should be smart enough to give `--delete_all=False` if we don't want it to delete everything.